### ISSUE TITLE
New Update Visualizer doesn't work

### ISSUE THREAD

**Issue by justkatia** [onlaj/Piano-LED-Visualizer#354]
Since I've updated the new version the Visualizer menu doesn't appear on my RPi screen anymore. I tried to look for solutions in the previous issues but none of them worked for me. I removed and reinstalled the Piano-LED-Visualizer but this doesn't work either. I would be thankful for your help :)

**Comment by onlaj** [comment]
Hello.
Can you connect to your RPi through SSH and manually start Visualizer?

Use those commands:
`sudo pkill python3` to stop working python script

`sudo python3 /home/Piano-LED-Visualizer/visualizer.py` to start script

If it shows any errors copy and paste them here.

**Comment by justkatia** [comment]
Yes ssh does work.  I tried and I have an import error: ImportError: cannot import name 'safe_join' from 'werkzeug.utils' (/usr/lib/python3/dist-packages/werkzeug/utils.py)

<img width="819" alt="Screen Shot 2022-05-04 at 13 16 10" src="https://user-images.githubusercontent.com/86355889/166672303-2a8e36f1-c700-4e12-9724-4e29400270ef.png">

**Comment by onlaj** [comment]
Hmm, it should be installed by default, but try
`sudo pip3 install Werkzeug`

**Comment by justkatia** [comment]
It says: Requirement already satisfied: Werkzeug in /usr/lib/python3/dist-packages (1.0.1)
<img width="573" alt="Screen Shot 2022-05-04 at 13 32 12" src="https://user-images.githubusercontent.com/86355889/166673405-eacb039c-73ed-4d1d-b48e-a2a01e558ec3.png">

**Comment by onlaj** [comment]
Ok, it seems you have older version installed, try
`sudo pip3 install Werkzeug --upgrade`

**Comment by justkatia** [comment]
thanks. apparently I could install a new version but now I get another import error if I run the commands xD
<img width="999" alt="Screen Shot 2022-05-04 at 13 43 23" src="https://user-images.githubusercontent.com/86355889/166675021-92723da5-c092-4101-b8b9-ac680c99885f.png">

**Comment by onlaj** [comment]
Maybe try to reinstall all required packages?
`cd /home/Piano-LED-Visualizer`
`sudo pip3 install -r requirements.txt --upgrade --force-reinstall`

**Comment by justkatia** [comment]
wow thank you :) it worked !

**Comment by BaseD75** [comment]
Great thank you, had the same issue

**Comment by vulcanjedi** [comment]
Thank you! Me too!
